testsuite/features/secondary/srv_cobbler_sync.feature

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Run Cobbler Sync via WebUI
+
+  Scenario: Login as admin
+    Given I am authorized for the "Admin" section
+
+  Scenario: Sanity check for page existance
+    When I follow the left menu "Admin > Manager Configuration > Cobbler"
+    Then I should see a "Uyuni Configuration - Cobbler. " text
+    And I should see a "Setup your Uyuni Cobbler settings below. " text
```

Comment:
You have to differentiate between SUMA and Uyuni here.

Reply:
Oh right completely forgot I will change this for the ports

Reply:
So how would I circumvent that? not check for the text at all? 🤔

Reply:
You can use tags for that, e.g. `@uyuni` and `@susemanager` but I am not sure if you can use it in scenarios directly or just directly above a scenario.

Reply:
I have only seen them used directly above scenarios ... I changed the text in the SUMA Ports but I'll take a closer look when I get to the office

Reply:
Do you need to repeat the scenario twice, one with a susemanager tag, and another with a uyuni tag, and then the different text on it.
That's the simplest way, the complex would be to change the text inside the step definition, but I would prefer the simple approach, as we are already using it in multiple palces.